src/igvfd/tests/data/inserts/treatment.json

```diff
@@ -0,0 +1,28 @@
+[
+    {
```

Comment:
I would like to see another 2-3 treatments, one that has an NTR, ones using different purposes, one using notes.  The inserts provide you with a way to test that all of your properties are working and gives you reasonable examples to like to biosamples.